lib/array_intersection.rb

```diff
@@ -1,6 +1,23 @@
 # Returns a new array to that contains elements in the intersection of the two input arrays
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2) because there are nested loops and n is the length of the input array
```

Comment:
The two arrays are not guaranteed to be of the same length.  This is more accurate to be O(n + m) where n and m are the lengths of the respective arrays.